package.json

```diff
@@ -73,6 +73,7 @@
     "bcrypt": "5.1.1",
     "bullmq": "4.17.0",
     "content-disposition": "0.5.4",
+    "csv-writer": "1.6.0",
```

Comment:
We are already using Papaparse in the project, maybe we can use the [`unparse`](https://www.papaparse.com/docs#json-to-csv) function to convert json data to csv ? Also the library proposed has not been updated since 4 years, and there are updates not released since 3 years, the project seems to be dead.